packages/oft-evm/contracts/NativeOFTAdapter.sol

```diff
@@ -112,7 +112,7 @@ abstract contract NativeOFTAdapter is OFTCore {
         // @dev Transfer tokens to the recipient.
         (bool success, ) = payable(_to).call{value: _amountLD}("");
         if (!success) {
-            revert CreditFailed();
+            revert CreditFailed(_to, _amountLD);
```

Comment:
Oh in my comment I meant something like this:

```solidity
error CreditFailed(address to, bytes revertData);

// If the call fails, `data` will hold the revert reason
(bool success, bytes memory data) = payable(_to).call{value: _amountLD}("");
if (!success) revert CreditFailed(_to, data);
```

Thoughts and prayers please @TRileySchwarz

Reply:
Oh i see! That would definitely be helpful -- I will update this in the next commit 😄

Reply:
I kept the `to` and `amountLD` parameters bc I think they could be helpful as well and also added the `revertData` as a third parameter now